File: Python3/537. Complex Number Multiplication.py
class Solution:
    def complexNumberMultiply(self, a, b):
        """
        :type a: str
        :type b: str
        :rtype: str
        """
        alist = a.split('+')
        blist = b.split('+')
        aa, ab = int(alist[0]), int(alist[1][: -1])
        ba, bb = int(blist[0]), int(blist[1][: -1])
        return str(aa*ba - ab*bb) + '+' + str(aa*bb + ab*ba) + 'i'

File: Golang/537. Complex Number Multiplication.go
func complexNumberMultiply(a string, b string) string {
    aslice := strings.Split(a, "+")
    bslice := strings.Split(b, "+")
    aa, _ := strconv.Atoi(aslice[0])
    ab, _ := strconv.Atoi(aslice[1][: len(aslice[1])-1])
    ba, _ := strconv.Atoi(bslice[0])
    bb, _ := strconv.Atoi(bslice[1][: len(bslice[1])-1])
    fmt.Println(aa, ab, ba, bb)
    return strconv.Itoa(aa*ba - ab*bb) + "+" + strconv.Itoa(aa*bb + ba*ab) + "i"
}
